README.md

```diff
@@ -1,15 +1,4 @@
 # flutter_weather_sensor_plugin
 
-A new flutter plugin project.
-
-## Getting Started
-
-This project is a starting point for a Flutter
-[plug-in package](https://flutter.dev/developing-packages/),
-a specialized package that includes platform-specific implementation code for
-Android and/or iOS.
-
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+Flutter wrapper of native weather sensor generation libraries.
```

Comment:
Optional: I haven't looked but is there anything in addition to this that is offered in the readme of the RN version? Or the Native?

Reply:
@derrikf-spindance @paigeb-spindance The only additional information that you would want to add would be any instructions on how to add the flutter library to an existing flutter project.

Reply:
added package installation instructions